box/box.py

```diff
@@ -411,13 +411,14 @@ def __box_config(self) -> Dict:
 
     def __recast(self, item, value):
         if self._box_config["box_recast"] and item in self._box_config["box_recast"]:
+            recast = self._box_config["box_recast"][item]
```

Comment:
Only added the additional var to keep the `if isinstance(...) and issubclass(...)` line from being too long. Let me know if you'd like to keep the lookup everywhere.